.travis.yml

```diff
@@ -9,6 +9,7 @@ services:
 - docker
 
 before_install:
+- kerl list installations
 - docker build -t build_ubuntu https://raw.githubusercontent.com/systemd/erlang-sd_notify/master/docker/ubuntu_18_3/Dockerfile
```

Comment:
yes, exactly. We had to do that for this reason: https://github.com/systemd/erlang-sd_notify/issues/9#issuecomment-229973764